app/controllers/healthcheck_controller.rb

```diff
@@ -26,4 +21,14 @@ def sendgrid_alive?
   def messagelabs_alive?
     SendgridHelper.smtp_alive?(*PrisonMailer.smtp_settings.values_at(:address, :port))
```

Comment:
I agree! I'm also not a fan of most of these 'Helper' module function things that litter the codebase.